docker-compose.yml

```diff
@@ -40,6 +40,7 @@ services:
       - JITSI_USER=fbs
       - JWT_SECRET=8Dsupersecurekeydf0
       - BBB_SECRET
+      - BBB_URI=http://host.docker.internal:8085
```

Comment:
For `host.docker.internal` to work under linux the following must be added to the `http` container in `docker-compose.yml`:

```yml
extra_hosts:
      - "host.docker.internal:host-gateway"
```